server/instance_cloud.go

```diff
@@ -74,9 +74,10 @@ func (jci jiraCloudInstance) GetJIRAClient(jiraUser JIRAUser) (*jira.Client, err
 		return client, nil
 	}
 
-	client, err = jci.getJIRAClientForServer()
+	//TODO decide if we ever need this as the default client
```

Comment:
I don't think we would want to fall back to using the server client. That seems dangerous as it likely has more permissions.